.github/workflows/ci-linux-incremental.yml

```diff
@@ -107,14 +110,6 @@ jobs:
 
   site:
     needs: [changed_files]
-    if: |
-      github.event_name != 'pull_request' ||
-      ((github.event.action != 'labeled' &&
-         (contains(github.event.pull_request.labels.*.name, 'c: packages: standard') ||
-          contains(github.event.pull_request.labels.*.name, 'c: packages: optional'))) ||
-       (github.event.action == 'labeled' &&
-         (github.event.label.name == 'c: packages: optional' ||
-          github.event.label.name == 'c: packages: standard')))
     uses: ./.github/workflows/docker.yml
```

Comment:
Does removing these lines mean that "CI Linux incremental" will run without adding "ci:" labels?

Reply:
Reacting to "label" events was a mechanism that never worked very well. It was removed from most jobs in this file already. (It's been replaced by the path filter at the top of the file and the changed-files mechanism.) 

These ones here for the "sitepackages" tetss were inexplicably left in - possibly by a mistake in merging.

Reply:
(This was done in #37222.)

Reply:
I see.